src/pages/contract/HistoryInfo.tsx

```diff
@@ -0,0 +1,28 @@
+import { ContractCodeHistoryEntry } from "@cosmjs/cosmwasm";
+import React from "react";
+
+import { CodeLink } from "../../components/CodeLink";
+
+interface Props {
+  readonly contractCodeHistory: readonly ContractCodeHistoryEntry[];
+}
+
+export function HistoryInfo({ contractCodeHistory }: Props): JSX.Element {
+  return (
+    <div className="card mb-3">
+      <ul className="list-group list-group-flush">
+        <li className="list-group-item">
+          <span title="The contract level message history">History</span>
+        </li>
+        {contractCodeHistory.map((entry) => (
+          <li className="list-group-item">
```

Comment:
Isn't React complaining about missing `key=` here?

Reply:
Oops, yes it is. But I don't know what to use as a unique key from `ContractCodeHistoryEntry` properties

Reply:
You can use the index via: `contractCodeHistory.map((entry, index) => (`